src/config/feature-flags.ts

```diff
@@ -29,7 +29,8 @@ export enum BooleanFlags {
 	USE_NEW_GITHUB_CLIENT_FOR_GET_REPO = "use-new-github-client-for-get-repo",
 	USE_NEW_GITHUB_CLIENT_FOR_INSTALLATION_API = "use-new-github-client-for-installation-api",
 	USE_NEW_GITHUB_CLIENT_FOR_REDIRECT = "use-new-github-client-for-redirect",
-	RETRY_ALL_ERRORS = "retry-all-errors"
+	RETRY_ALL_ERRORS = "retry-all-errors",
+	GHE_SERVER = "ghe_server",
```

Comment:
Renamed the feature flag. This is going to be a 'global' flag the team is going to use to wrap all GHE server changes.